# Dynamic Planner Default Params
main_run_freq                           : 2.0
voxel_dim                               : 0.15  # Unit: meter
robot_dim                               : 0.6   # Unit: meter
sensor_range                            : 10.0  # Unit: meter
visualize_ratio                         : 0.5
ceil_height                             : 5.0   # Unit: meter
is_static_env                           : true
world_frame                             : map

# Map Handler Params
MapHandler/cell_height                  : 2.0     # Unit: meter
MapHandler/cell_length                  : 5.0     # Unit: meter
MapHandler/max_length                   : 1000.0  # Unit: meter

# OMPL Planner Params
OMPLPlanner/planner_step                : 2.0     # Unit: meter
OMPLPlanner/collision_radius            : 0.5     # Unit: meter
OMPLPlanner/converage_dist              : 2.5     # Unit: meter
OMPLPlanner/project_dist                : 5.0     # Unit: meter

# Dynamic Planner Utility Params
Util/collide_counter                    : 2
